docs/guidelines/content/set-plagiarism.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Set Plagiarism
+
```

Comment:
Please remove this line. Frontmatter at the top of documents is structured data that is expecting an exact format -- this newline could wind up breaking the document at buildtime.